drop-analytics/src/lib.rs

```diff
@@ -19,11 +19,11 @@ const MOOSE_STATUS_SUCCESS: i32 = 0;
 const MOOSE_VALUE_NONE: i32 = -1;
 
 #[derive(Clone, Copy, Debug, Serialize, Deserialize)]
-pub enum Phase {
-    #[serde(rename = "start")]
-    Start,
-    #[serde(rename = "end")]
-    End,
+pub enum TransferDirection {
+    #[serde(rename = "upload")]
+    Upload,
+    #[serde(rename = "download")]
+    Download,
```

Comment:
We have 4 different combinations: upload can start/stop, and the same for download. However we only had two(start, end), now we have two as well but different(upload, download). My concern is that we effectively lose some information that might be desired by analytics team. If we send two events(on start and end) then how do analytics differentiate between the two?